.gitignore

```diff
@@ -9,3 +9,8 @@ dist
 skepticoin/scmversion.py
 
 __pycache__
+chain 
+wallet.json 
+peers.json 
+.vscode 
+venv 
```

Comment:
Please remove `.vscode` here, I was asked the same in an older PR.
Not sure if we want to keep `venv` here either. @sashimi-houdini 
IMO it's justified to keep the other 3 added items.

Reply:
@sashimi-houdini any comment on this? I would prefer to leave `.vscode` in the `.gitignore` so that I don't have to remember to manually remove the .vscode folder from every commit. That would make it harder to contribute. I can remove the `venv` since that can live easily outside of the source foder.

Reply:
> so that I don't have to remember to manually remove the .vscode folder from every commit.

You can use `git commit -a` to commit all already-tracked files. If you want to track additional files you can do `git add <file>`.
So if you don't track the `.vscode` folder, you never have to manually remove it either.

Reply:
Updated.